main/icicle-repl.hs

```diff
@@ -189,12 +192,20 @@ handleLine state line = case readCommand line of
 
   Just (CommandLoad fp)      -> do
     s  <- liftIO $ T.readFile fp
-    case SR.readFacts dict s of
+    case SR.readFacts (dictionary state) s of
       Left e   -> prettyHL e >> return state
       Right fs -> do
         HL.outputStrLn $ "ok, loaded " <> fp <> ", " <> show (length fs) <> " rows"
         return $ state { facts = fs }
 
+  Just (CommandLoadDictionary fp) -> do
+    s  <- liftIO $ T.readFile fp
+    case SR.readDictionary s of
+      Left e   -> prettyHL e >> return state
+      Right ds -> do
+        HL.outputStrLn $ "ok, loaded " <> fp
```

Comment:
it's silly but maybe just something like "loaded dictionary with 5 fields".
if CommandSetShow showed the dictionary too it'd be super neato